### ISSUE TITLE
optional argument for Packaging is shown as not allowed property

### ISSUE THREAD

**Issue by BPylypenko** [odahu/odahu-flow#368]
**Describe the bug**
In invalid_argument_cli.yaml passed invalid type for `dockerfileAddCondaInstallation` key. `str` instead of `bool`.

`odahuflowctl --verbose pack create -f ${RES_DIR}/arguments/invalid_argument_cli.yaml`
![image](https://user-images.githubusercontent.com/49116723/89533346-f5462700-d7fb-11ea-8f6d-9d5702e525a4.png)

**Exp. Behavior**
The error should resemble this one:
![image](https://user-images.githubusercontent.com/49116723/89533453-21fa3e80-d7fc-11ea-8f50-34456584bddd.png)
and as said in docs (https://docs.odahu.org/ref_packagers.html), this property is allowed.

**Comment by keshamin** [comment]
@BPylypenko please provide a packaging file with an example.

**Comment by keshamin** [comment]
While investigating we found out that `dockerfileAddCondaInstallation` field (as well as `dockerfileCondaEnvsLocation`) are not used in packagers  anymore (both `docker-cli` and `docker-rest` packagers). These arguments were removed in this commit: https://github.com/odahu/odahu-packager/commit/739bc1ccebe0e20015cf70fb4bfddce99ed10753, but docs were not updated. So now we just need to actualize docs.
